src/index.js

```diff
@@ -1,28 +1,28 @@
-export { default as Button } from './buttons/Button'
-export { default as WrappedButton } from './buttons/WrappedButton'
-export { default as CopyToClipboard } from './buttons/CopyToClipboard'
-export { default as FigureCard } from './buttons/FigureCard'
-export { default as Forward } from './buttons/Forward'
-export { default as Dialog } from './overlays/Dialog'
-export { default as BottomDialog } from './overlays/BottomDialog'
-export { default as Field } from './forms/Field'
-export { default as PhoneField } from './forms/PhoneField'
-export { default as BankAccountField } from './forms/BankAccountField'
-export { default as Selector } from './forms/Selector'
-export { default as SelectionList } from './forms/SelectionList'
-export { default as Toggle } from './forms/Toggle'
-export { default as ActionItem } from './lists/ActionItem'
-export { default as ListItem } from './lists/ListItem'
-export { default as ContentItem } from './lists/ContentItem'
-export { default as Spinner } from './loading/Spinner'
-export { default as Stepper } from './navigation/Stepper'
-export { default as Screen } from './structure/Screen'
-export { default as Scroll } from './structure/Scroll'
-export { default as Wrapper } from './structure/Wrapper'
-export { default as Heading } from './typography/Heading'
-export { default as Text } from './typography/Text'
-export { default as TextField } from './forms/TextField'
-export { default as TextFieldRule } from './forms/TextFieldRule'
-export { default as TopBar } from './navigation/TopBar'
-export { default as DateField } from './forms/DateField'
-export { default as CopyableList } from './lists/CopyableList'
+export { default as Button } from './buttons/Button/Button';
```

Comment:
Why did you change the order? Is this supposed to have a special sort?

Reply:
Nope, it was only to group them a bit by "folder" (forms, lists, etc...)